flocx_market/db/sqlalchemy/offer_model.py

```diff
@@ -0,0 +1,16 @@
+from flocx_market.db.orm import orm
+import sqlalchemy_jsonfield
+
+
+class OfferModel(orm.Model):
+    __tablename__ = 'offers'
+    marketplace_offer_id = orm.Column(orm.String(64), primary_key=True, autoincrement=False)
+    provider_id = orm.Column(orm.String(64), nullable=False)
+    creator_id = orm.Column(orm.String(64), nullable=False)
+    marketplace_date_created = orm.Column(orm.DateTime(timezone=True), nullable=False)
+    status = orm.Column(orm.String(15), nullable=False, default='available')
+    server_id = orm.Column(orm.String(64), nullable=False, unique=True)
+    start_time = orm.Column(orm.DateTime(timezone=True), nullable=False)
+    end_time = orm.Column(orm.DateTime(timezone=True), nullable=False)
+    server_config = orm.Column(sqlalchemy_jsonfield.JSONField(enforce_string=True, enforce_unicode=False), nullable=False)
```

Comment:
I've had more success with this sort of field as an oslo JsonEncodedDict, as in
https://github.com/CCI-MOC/esi-leap/blob/master/esi_leap/db/sqlalchemy/models.py#L57